doc/specifications/rest/REST-API-V2.rst

```diff
@@ -7076,6 +7139,37 @@ ContentCreate XML Schema
       <xsd:element name="ContentCreate" type="contentCreateType"></xsd:element>
     </xsd:schema>
 
+
+.. _SimplifiedContentCreate:
+
+Simplified ContentCreate Example
+--------------------------------
+
+.. code:: json
+
+    {
+        "ContentCreate": {
+            "ContentType": "/content/types/_by_identified/article",
+            "LocationCreate": {
+                "_parentUrlAliasPath": "/content/urlaliases/_by_path/places-tastes/tastes/"
+            },
+            "ContentSection": "/content/section/_by_identifier/places",
+            "Owner": "/user/users/_by_login/jessica",
+            "alwaysAvailable": "false",
+            "modificationDate": "2017-03-02T12:00:00",
+            "remoteId": "remote123456",
+            "fields": {
+                "en-US": {
+                    "title": "This is a title",
+                    "intro": {
+                        "xhtml5edit": "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<section xmlns=\"http://ez.no/namespaces/ezpublish5/xhtml5/edit\"><p>Article intro.</p></section>\n",
+                        "xml": "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<section xmlns=\"http://docbook.org/ns/docbook\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" xmlns:ezxhtml=\"http://ez.no/xmlns/ezpublish/docbook/xhtml\" xmlns:ezcustom=\"http://ez.no/xmlns/ezpublish/docbook/custom\" version=\"5.0-variant ezpublish-1.0\"><para>Article intro.</para></section>\n"
```

Comment:
@tobyS do you imply by this, that both `xhtml5edit` and `xml` values are required or just one of them depending on API consumer needs? I'm asking because if we want to simplify, only one should be required, but both supported (just IMHO).